stripe/src/main/java/com/stripe/android/view/CountryUtils.kt

```diff
@@ -1,7 +1,10 @@
 package com.stripe.android.view
 
+import com.stripe.android.model.CountryCode
 import java.util.Locale
 
+internal fun Locale.getCountryCode(): CountryCode = CountryCode.create(this.country)
```

Comment:
Nice! When I look at `Locale#getCountry()` I see that the description:

```
Returns the country/region code for this locale, which should
either be the empty string, an uppercase ISO 3166 2-letter code,
or a UN M.49 3-digit code.
```

Do you know when this returns a 3-digit code?

Reply:
If this can return a 3-digit code, should we also rename `CountryCode`'s `twoLetters` property to be more generic? Maybe just `value` or `code`?

Reply:
UN M.49 3-digit codes are for regions, not countries, and update the variable to value.  

I think I will also move this extension function to CountryCode